p5/core/transforms.py

```diff
@@ -176,20 +159,19 @@ def apply_matrix(transform_matrix):
     :param transform_matrix: The new transform matrix.
     :type transform_matrix: np.ndarray (or a 4×4 list)
     """
-    tmatrix = np.array(transform_matrix)
-    p5.renderer.transform_matrix = p5.renderer.transform_matrix.dot(tmatrix)
+    p5.renderer.apply_matrix()
 
 
 def reset_matrix():
     """Reset the current transform matrix.
     """
-    p5.renderer.transform_matrix = np.identity(4)
+    p5.renderer.reset_matrix()
 
 
 def print_matrix():
     """Print the transform matrix being used by the sketch.
     """
-    print(p5.renderer.transform_matrix)
+    print(p5.renderer.print_matrix())
```

Comment:
I think it's a little odd for `renderer.print_matrix` to not print the matrix but return a string instead. I think we could either make the renderer function print directly, or rename the renderer function.

Reply:
Fixed